### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by agmalis**
New section 5, and capitalized headings.

### CODE REVIEW

File: draft-ietf-detnet-dp-sol-ip-02.xml [hunk 1]
```diff
@@ -759,76 +759,136 @@ DetNet                    |L2/SbN|          |L2/SbN|
...
+    combination thereof. Specific considerations and requirements for the DetNet Controller Plane are discussed in <xref target="control-management-requirements"/>.
+ </t>
+  <section title="Flow Identification and Aggregation">
+   <t>
+   <xref target="sec_dt_dp"/> introduces the use of the IP "6-tuple" for flow identification, and <xref target="QoS"/> goes on to discuss how identified flows use specific QoS mechanisms for flow-specific traffic treatment, including path control and resource allocation. <xref target="ip-flow-id"/> contains detailed DetNet IP flow identification procedures. Flow identification will play an important role for the DetNet controller plane.
+ </t>
+     <t>
+      <xref target="aggregation"/> and <xref target="agg-con"/> discuss the use of flow aggregation in DetNet. Flow aggregation can be accomplished using any of the 6-tuple fields defined in <xref target="ip-flow-id"/>, using a DSCP identified traffic class or other mechanisms. It will be the responsibility of the DetNet controller plane to be able to properly provision the use of these aggregation mechanisms. These requirements are included in <xref target="control-management-requirements"/>.
```
Comment (louberger) [inline draft-ietf-detnet-dp-sol-ip-02.xml]: nit: how about s/other mechanism/other field

**Comment by agmalis** [comment]
Lou,

Yes, I'll make that change.

Cheers,
Andy

**Comment by agmalis** [comment]
And done.

**Comment by louberger** [comment]
Thanks....

Lou
PS you might want to turn off line wrap in your editor...


----------

**Comment by agmalis** [comment]
Lou,

I just checked the settings and I don't believe it's doing any hard
wrapping. What are you seeing?

Cheers,
Andy

**Comment by louberger** [comment]
right - it's best to use hard wrapping for maximum editor independence 
as well as github reviews.

Lou

File: draft-ietf-detnet-dp-sol-ip-02.xml [hunk 2]
```diff
@@ -759,76 +759,136 @@ DetNet                    |L2/SbN|          |L2/SbN|
...
+   <t>The ability to support DetNet flow aggregation</t>
+   <t>Advertise static and dynamic node and link resources such as capabilities and adjacencies to other network nodes (for dynamic signaling approaches) or to network controllers (for centralized approaches)</t>
+   <t>Scale to handle a large number of DetNet flows in a domain (which may require per-flow signaling or provisioning)</t>
+   <t>Provision flow identification information at each of the nodes along the path, and it may differ depending on the location in the network and the DetNet functionality.</t>
+  </list> </t>
+  <t>
+  These requirements, as stated earlier, could be satisfied using distributed control protocol signaling, centralized network management provisioning mechanisms, or hybrid combinations of the two, and could also make use of IPv6-based segment routing.
+  </t>
```
Comment (vb-balazs) [inline draft-ietf-detnet-dp-sol-ip-02.xml:883]: Why to name here IPv6 SR explicitly, but no other possible solutions?

**Comment by agmalis** [comment]
Balázs,

This is similar to the text in the MPLS document, but that document
references SR-MPLS. In the IP case, I figured that SR-MPLS was off the
table, so I changed the reference to SRv6. The earlier part of the sentence
already references control plane signaling, centralized management, and
hybrid solutions. In addition to those, I wanted to include the fact that
SRv6 is also a valid solution here. If there's anything else that you would
like to include, we can certainly add that.

Cheers,
Andy

**Pull request merged**